src/app/components/Footer.tsx

```diff
@@ -8,15 +8,15 @@ const Footer: React.FC = () => {
 
   const email = 'womencodersnetwork@gmail.com';
   return (
-    <footer className="w-full p-10 bg-dark-violet text-leafy font-main">
-        <div className="flex items-center justify-around">
-            <button className="h-min bg-leafy text-dark-violet p-2 rounded border-2 border-solid border-[#000]-800 shadow-xl text-xl font-bold">
+    <footer className='w-full p-10 bg-dark-violet text-leafy font-main'>
+        <div className='flex justify-between max-[640px]:flex-col'>
```

Comment:
Why are you using justify-between instead of justify-around? Did you forget to do a `git pull` before updating?

Reply:
@sunsplat synced mobile layout pr with updated dev branch